<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm700</class>
 <widget class="QWidget" name="LoginForm700">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#LoginForm700{border-image: url(:/image/login.png);}</string>
  </property>
  <widget class="QLabel" name="label_swv">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>--------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_msg">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>--------------------</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_psw">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_user">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>密  码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_weight">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hdv">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>--------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tare">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
